src/modules/dashboard/components/ActionsPage/StakingValidationError/StakingValidationError.tsx

```diff
@@ -40,11 +42,35 @@ const stakeValidationMSG = defineMessage({
 
 const displayName = 'StakingValidationError';
 
-const StakingValidationError = ({ stakeType, errorValues }: Props) => (
-  <div className={styles.validationError}>
-    <FormattedMessage {...stakeValidationMSG[stakeType]} values={errorValues} />
-  </div>
-);
+const StakingValidationError = ({ stakeType, errorValues }: Props) => {
+  const { setIsOpen: openTokenActivationPopover } = useContext(
+    TokenActivationContext,
+  );
+
+  if (stakeType === 'tokens') {
+    return (
+      <Button
+        text={stakeValidationMSG.tokens}
+        textValues={{
+          leftToActivate: errorValues?.leftToActivate as PrimitiveType,
+          tokenSymbol: 'A',
```

Comment:
Just noticed, this should be dynamic right?

Reply:
great catch! leftover from the working phase :) fixed now